src/c/sk_bitmap.cpp

```diff
@@ -300,3 +320,9 @@ sk_colortable_t* sk_bitmap_get_colortable(sk_bitmap_t* cbitmap)
 {
     return ToColorTable(AsBitmap(cbitmap)->getColorTable());
 }
+
+void sk_bitmap_set_pixels(sk_bitmap_t* cbitmap, void* pixels, sk_colortable_t* ctable)
+{
+    SkBitmap* bmp AsBitmap(cbitmap);
```

Comment:
Oops. I didn't mean to push these changes to my public repo. I'm still debugging a few issues with the bindings.